src/main/java/lotto/domain/rank/Rank.java

```diff
@@ -0,0 +1,37 @@
+package lotto.domain.rank;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public enum Rank {
+    OUT(0, 0L),
+    SIXTH(1, 0L),
+    FIFTH(2, 0L),
+    FOURTH(3, 5000L),
+    THIRD(4, 50000L),
+    SECOND(5, 1500000L),
+    FIRST(6, 2000000000L);
```

Comment:
언더스코어를 통해 가독성을 높일 수 있습니다 :)

```suggestion
    FIRST(6, 2_000_000_000L);
```